### DISCUSSION TITLE
The eformsSdkVersion has an invalid format, please respect the following format eforms-sdk-x.x SDK 1.9

### DISCUSSION THREAD

**Discussion by jaggaer-fgeniola** [OP-TED/eForms-SDK#753] [Q&A] [answered]
Hi,
since this morning we have been receiving this error message in the test environment:
**400 : "{"errorCode":400,"message":"The eformsSdkVersion has an invalid format, please respect the following format eforms-sdk-x.x","timestamp":"2023-11-08T09:43:49Z"}"**

The data reported in the xml is:
**<cbc:CustomizationID xmlns="">eforms-sdk-1.9</cbc:CustomizationID>**

We don't understand where the problem is, could you tell us the solution?

Thanks

**Comment by YvesJo** [comment, marked as answer]
Hi,
For this incident, please get directly in touch with the helpdesk ted@publications.europa.eu
